Client/src/Remote/Endpoints/VariableEndpoint.tsx

```diff
@@ -0,0 +1,16 @@
+import { get } from "../FluentRequest"
+
+import SnackbarUtils from "../../Components/Utils/SnackbarUtils"
+import { IVariableNameModel } from "../../Models/IVariableNameModel"
+
+const variableRoute = '/api/v1/variables'
+
+export const callGetVariables = async (): Promise<IVariableNameModel> => {
+  const result = await get(variableRoute).json()
+  if (result as IVariableNameModel[]) {
```

Comment:
Nice check here. Did not know this worked!

Reply:
i dont think this does what you think. 'as' is just a cast, but it doesnt add any validation. its the same as if(result)

Reply:
Yea since they are interfaces it's not easy to type cast them in conditions